src/components/TimeSeriesCard/TimeSeriesCard.story.jsx

```diff
@@ -683,7 +683,7 @@ storiesOf('Watson IoT/TimeSeriesCard', module)
             zoomBar: {
               enabled: true,
               axes: 'top',
-              // initialZoomDomain: []
+              view: select('view', ['slider_view', 'graph_view'], 'slider_view'),
```

Comment:
```suggestion
              view: select('zoomBar.view', ['slider_view', 'graph_view'], 'slider_view'),
```

Reply:
I think this may be happening because of all of the memoization in the card. I went ahead and made 2 separate stories instead

Reply:
This is actually a bug. Looking into it now

Reply:
Great find. I was actually calling the improper prop. I also realized that there's already an `content` object knob where the view can be changed

Reply:
Honestly I liked the old knob setup because it showed the options instead of being a freeform text input with no context as to what the options are, but it does flip the type of zoombar now 👍